src/utils/boltzClient.ts

```diff
@@ -3,18 +3,22 @@ import { Musig } from "boltz-core";
 import { Buffer } from "buffer";
 import { Transaction as LiquidTransaction } from "liquidjs-lib";
 
+import { config } from "../config";
+import { SwapType } from "../consts/Enums";
 import { fetcher } from "./helper";
 
+const cooperativeErrorMessage = "Cooperative swaps are disabled";
```

Comment:
What are "cooperative swaps"?
```suggestion
const cooperativeErrorMessage = "cooperative signatures for swaps are disabled";
```